.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature Request
+about: A suggestion for a new feature
+labels: enhancement
+---
+
+<!--
+	Thank you for contributing to SwiftSyntax!
+	Before you submit your issue, please replace each paragraph
+	below with information about your proposed feature.
+-->
+
+### Feature Request: _<Feature Name>_
+
+#### Description
+
+Replace this paragraph with a description of your proposed feature.
+Sample documentation catalogs that show what's missing,
+or what new capabilities will be possible, are very helpful!
+Provide links to existing issues or external references/discussions, if appropriate.
```

Comment:
I think this is also still docc-specific.

Reply:
Made it a bit more generic.